### PR TITLE
fix: summary tag in rss feed now renders formatted html

### LINKED ISSUE #3763: RSS feed unformatted and not full text
**Describe the bug**
Unlike the RSS feed for the Mozilla blog (https://blog.mozilla.org/), the blog of the Foundation (https://foundation.mozilla.org/en/blog/) does not provide a full text RSS feed and the texts provided are not formatted (no line breaks, italics, links etc.)

**To Reproduce**
Subscribe to https://foundation.mozilla.org/en/blog/atom/ or https://foundation.mozilla.org/en/blog/rss/ using Feedly. I haven't tried other RSS services.

**Screenshots**
Here is a comparison of a blog post that appeared both on the [blog of the Foundation](https://foundation.mozilla.org/en/blog/the-next-internet-health-report-will-zoom-in-on-solutions/) and [at internethealthreport.org](https://internethealthreport.org/the-next-internet-health-report-will-zoom-in-on-solutions/), both displayed in Feedly:

Screenshot 2019-10-10 at 15 34 31

**Additional context**
When you try to add the feed to Feedly by just using https://foundation.mozilla.org/en/blog/, Feedly will fail to find any RSS feed saying "Please contact the website and ask them if they offer a valid RSS feed." Not sure if this is a bug in Feedly or a problem with the Foundation's website. Using https://foundation.mozilla.org/en/blog/atom/ or https://foundation.mozilla.org/en/blog/rss/ works.

### PR DESCRIPTION
**Pull request by oxalorg**
Fixes #3763 

This PR makes RSS content to be rendered as HTML to allow proper formatting in RSS readers. Previously it was being rendered as Plain Text.

The culprit was the call to `parsed.get_text()[:1000]`, that function was converting the html to a string. I believe this was used so as to truncate easily (truncating HTML is a bit difficult).

Thankfully, Django has an inbuilt Truncator for HTML, which I have used.

I have also removed the use of `BeautifulSoup()` as we get the HTML directly from the body field.

### Before This PR
image

Source XML
```
<description>Begin process provide we bag movement home.Officer crime resource give summer condition adult. Middle imagine teacher peace month short.
Want budget evidence it. Walk manage bill under step. Whole executive travel human sit him. This sentence is in bold text. Game standard walk space. People them least reflect receive report.
Congress authority see student. Admit ten man process. This is a link to a fake url! Trade lose age father note deal cold. Instead church believe record recent appear site. Amount wear bad author those. Push building red design rest. Occur first financial speak far. Instead poor skill Mr. History off method let investment expect create perform. Under ready above. World west yeah suddenly fund. Significant music sure event service west onto. Image evidence situation thank time candidate. Woman able have rest but serious argue. Player small church we cover billion popular. Natural food eye. Gas director enter view TV recently window necessary. Position article reach[...]</description>
```

### After this PR

image

Source XML:
```
<description>
<div class="block-paragraph"><div class="rich-text"><h3><a class="fragment-id" id="begin-process-provide-we-bag-movement-home"></a>Begin process provide we bag movement home.</h3><p>Officer crime resource give <i>summer</i> condition adult.</p><p>Middle imagine teacher peace month short. <b>Want budget evidence it.</b></p><p><i>Walk manage bill under step</i>. Whole executive travel human sit him. <b>This sentence is in bold text.</b> Game standard walk space.</p><p>People them least reflect receive report. Congress authority see student. Admit ten man process. <a href="https://pena.com/">This is a link to a fake url!</a> Trade lose age father note deal cold. Instead church believe record recent appear site. Amount wear bad author those. Push building red design rest. Occur first financial speak far. Instead poor skill Mr. History off method let investment expect create perform. Under ready above. World west yeah suddenly fund. Significant music sure event service west onto. Image evidence situation thank time candidate. Woman able have rest but serious argue. Player small church we cover billion popular. Natural food eye. Gas director enter view TV recently window necessary. Position article reach c…</p></div></div>
</description>
```

## How did I test this locally?

Thunderbird of course 😊! Set up a new feeds account, and then add the localhost dev URL as a new subscription.

To refresh the messages after a change, delete the subscription and add it again. (let me know if there's a better way to refresh in Thunderbird haha)

## Possible Issues

The original issue mentions that the RSS feed does not render the full article. Right now the PR does not attempt to fix this and instead truncates HTML like the previous version does.

If you think we should show the entire content of the article, let me know and I will update this PR. 😃  

Also: Hi 👋 everyone, this would be my first contribution to this project. I saw this issue as a `good-first-issue` and picked it up. I have quite a bit of experience with Django, and have also worked a bit with Wagtail previously! ^_^

### CODE REVIEW

File: network-api/networkapi/wagtailpages/rss.py [hunk 1]
```diff
@@ -37,10 +36,7 @@ def item_link(self, item):
     def item_description(self, item):
         page = item.specific
         html = str(page.body)
-        parsed = BeautifulSoup(html, 'html.parser')
-        text = parsed.get_text()[:1000]
-        if len(text) == 1000:
-            text = f'{text}[...]'
+        text = Truncator(html).chars(1000, html=True)
```
Comment (oxalorg) [inline network-api/networkapi/wagtailpages/rss.py:39]: On 2nd thought, I should have renamed `text` to something more descriptive, something like `description` 🙈 ?

**Comment by Pomax** [comment]
Hi @oxalorg, thanks so much for doing this! It will have to sit for a few days because of the Easter holidays, but I'll look this over on Monday and hopefully there won't be any need for changes!

**Comment by oxalorg** [comment]
Hey, @Pomax thanks for the quick reply. No issues at all, and Happy Easter! 😄

File: network-api/networkapi/wagtailpages/rss.py (hunk 1 above)
Comment (Pomax) [reply network-api/networkapi/wagtailpages/rss.py:39]: keeping the name is fewer changes =)

**Comment by oxalorg** [comment]
Hey @Pomax thanks for merging this. I'll be having some free time over next week, so I'd love to contribute more. 

Are there any issues in which help is needed? (Currently, the `help-wanted` and `good-first-issue` labels are empty 🙈 )

**Pull request merged**